modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -119,6 +122,9 @@ GrainTracker::finalize()
                                                 _unique_grain_to_ebsd_num[grain_pair.first] :
                                                 grain_pair.first,
                                                 grain_pair.second._var_idx);
+
+          auto elem_it_pair = _elemental_data_2.emplace(elem_id, std::vector<unsigned int>(_n_vars, libMesh::invalid_uint));
+          elem_it_pair.first->second[grain_pair.second._var_idx] = _ebsd_reader ? _unique_grain_to_ebsd_num[grain_pair.first] : grain_pair.first;
```

Comment:
@friedmud - lost your original comment with the force push. The reason that I'm doing this emplace is because sometimes it inserts and sometimes it doesn't. I certainly could construct the vector and move it but I still need to update it multiple times as I hit an existing element again (overlap). These two lines are awesome because I _always_ get the iterator, but I sometimes get the new insertion.